docs/querying/api.md

```diff
@@ -382,6 +382,79 @@ $ curl http://localhost:9090/api/v1/alertmanagers
 }
 ```
 
+## Status
+
+Following status endpoints expose current Prometheus configuration.
+
+### Config
+
+The following endpoint returns currently loaded configuration file:
+
+```
+GET /api/v1/status/config
+```
+
+Currently, config is returned as dumped YAML file.
+
+```json
+$ curl http://localhost:9090/api/v1/status/config
+{
+  "status": "success",
+  "data": {
+    "yaml": "<content of the loaded config file in YAML>",
+  }
+}
+```
+
+### Flags
+
+The following endpoint returns flag values that Prometheus was configured with:
+
+```
+GET /api/v1/status/flags
+```
+
+All values are in a form of "string".
+
+```json
+$ curl http://localhost:9090/api/v1/status/flags
+{
+  "status": "success",
+  "data": {
+    "alertmanager.notification-queue-capacity": "10000",
```

Comment:
There's no need to list all of the flags here, just a small sampling will do.

Reply:
Not sure I understand. Why would we "sample" the configured flags we are showing.

Reply:
I meant only in the docs, otherwise we get to update this every time we get new flags.

Reply:
I can agree with @brian-brazil here, it is just an example.